### ISSUE TITLE
Installed extension, not seeing any warnings

### ISSUE THREAD

**Issue by allinws** [teledemic/vscode-branch-warning#18]
I've tried restarting vscode, and putting the following in my settings.json

    "branchwarnings.protectedBranches": [ "main", "master", "master-dockerize", "releases/**" ], 

But nothing happens and I don't see any warnings.

I'm running vscode 1.63.2 and git branch warnings 1.0.10

**Comment by allinws** [comment]
It seems like it don't work on my larger repository, but works fine on my smaller one. Any reason for this?

**Comment by lvlaanderen** [comment]
This extension works great if I open a repo as a folder, but nothing happens when I open it as a workspace?
This also doesn't work if I open a subfolder within a repo
using vscode 1.66.0 and git branch warnings 1.0.10

**Comment by teledemic** [comment]
PRs welcome!

**Comment by KostaProsenikov** [comment]
> This extension works great if I open a repo as a folder, but nothing happens when I open it as a workspace? This also doesn't work if I open a subfolder within a repo using vscode 1.66.0 and git branch warnings 1.0.10

Yes, ite seems it does not work once you have opened a subfolder from a Repo. Can this be fixed please ?
